tscat/__init__.py

```diff
@@ -67,6 +67,9 @@ def variable_attributes_as_dict(self) -> dict:
                 ret[k] = v
         return ret
 
+    def fixed_attributes_as_dict(self) -> dict:
```

Comment:
remove the _as_dict and I merge it :)

Reply:
While I'm at it, should I also change the `variable_attributes_as_dict()` to `variable_attributes()` ?

Reply:
Yes good idea :)